### DISCUSSION TITLE
GAMES ON RUFFLE

### DISCUSSION THREAD

**Discussion by CIACIACIACIACIACIA9** [ruffle-rs/ruffle#9814] [General] [answered]
Recently i visited https://www.gamesflow.com/ and urban basketball game shows introduction screen and play music but doesn't progress further.
Also on https://www.snokido.com/game/urban-basketball.

**Comment by n0samu** [comment]
Ruffle doesn't support that game yet. You can play it in Flashpoint: https://bluemaxima.org/flashpoint/

**Comment by CIACIACIACIACIACIA9** [comment, marked as answer]
Unfortunately many systems and mine also don't support flashpoint (maximus the name).

**Reply by n0samu** [reply]
In that case, you can look into restoring Flash Player on your computer. We have some info about that in our FAQ on Discord.

**Comment by CIACIACIACIACIACIA9** [comment]
Unfortunately we don't have access to that network (discord) for some reasons could you describe by what's the meaning of "restoring Flash Player"?.

**Reply by n0samu** [reply]
I've copied over the instructions from Discord to here: https://gist.github.com/n0samu/06bb20c24a67df4b3e39b3aeca44a86a

**Comment by CIACIACIACIACIACIA9** [comment]
I have seeing that urban basketball is now playable for the tournament but i think that the appearances can't change by the user.
I have seeing also that thundercats: the orb of madness is playable (introduction screen) but needs improvement.

**Comment by CIACIACIACIACIACIA9** [comment]
I have seeing that the game thundercats:the orb of madness is more playable and i almost reached to
the end.
Concerning the flash method of playing i recommend ruffle and lightspark(the new version) till are perfected and not the other methods thank you.

**Comment by CIACIACIACIACIACIA** [comment]
Good news are for now because i saw that thundercats:the rescue is fully playable with web extension of 07-06-2023 date.

**Comment by CIACIACIACIACIACIA** [comment]
Good news again for URBAN BASKETBALL game the appearances of the teams are correct now since 26-6-2023 update.
